LinkLiteAgent/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "linkliteagent"
+version = "0.1.0"
+license = "MIT"
+description = ""
+authors = [
+    "Daniel Lea <pszdl@exmail.nottingham.ac.uk>",
+    "Vasiliki Panagi <uizvp@exmail.nottingham.ac.uk>",
+    "Samuel Adejumo <uizsa2@exmail.nottingham.ac.uk>",
+    "Jonathan Couldridge <uczjc@exmail.nottingham.ac.uk>",
```

Comment:
Can this use the normal email format, since that's in my git config and linked to my GitHub account ;)

`Jonathan.Couldridge@nottingham.ac.uk`